docs/docs/features/libraries.md

```diff
@@ -40,6 +40,8 @@ In all above scan methods, Immich will check if any files are missing. This can
 
 Finally, files can be deleted from Immich via the `Remove Offline Files` job. This job can be found by the three dots menu for the associated external storage that was configured under Administration > Libraries (the same location described at [create external libraries](#create-external-libraries)). When this job is run, any assets marked as offline will then be removed from Immich. Run this job whenever files have been deleted from the file system and you want to remove them from Immich.
 
+In order to remove offline files that linked to spesifce paths, remove paths -> Scan for new files -> Then run remove offline files
```

Comment:
What is this trying to say?

Reply:
> What is this trying to say?

the correct order for deletion
https://discord.com/channels/979116623879368755/994044917355663450/1204154596067385354

Reply:
I think the same is already noted at the start of this section, right?

> Note: Either a manual or scheduled library scan must have been performed to identify offline assets before this process will work.

Reply:
It seems so :)
I missed it